server/src/main/java/org/opensearch/extensions/action/ExtensionsActionRequest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+
+public class ExtensionsActionRequest extends ActionRequest {
+    String firstRequest;
```

Comment:
Leftover from darft?

Reply:
Or is this a draft PR?  

Why is the first request so important to preserve?

Reply:
My bad, looks like I missed a commit during the rebase. 
This is moved to `action` which the extension would like to register.